runtime/caml/sizeclasses.h

```diff
@@ -3,17 +3,54 @@
 #define POOL_HEADER_WSIZE 4
 #define SIZECLASS_MAX 128
 #define NUM_SIZECLASSES 32
+
+/* The largest size for this size class.
+   (A gap is left after smaller objects) */
 static const unsigned int wsize_sizeclass[NUM_SIZECLASSES] =
-{ 1, 2, 3, 4, 5, 6, 7, 8, 10, 12, 14, 16, 17, 19, 22, 25, 28, 32, 33, 37, 42,
-  47, 53, 59, 65, 73, 81, 89, 99, 108, 118, 128 };
+{ /*  0:*/ 1, 2, 3, 4, 5
```

Comment:
I think this is missing end-of-line commas, causing the CI to fail parsing... :thinking:

Reply:
Ah... "Readable" is nice, but "Parseable" is definitely a plus.